src/main/java/com/mixpanel/android/viewcrawler/ViewCrawler.java

```diff
@@ -615,6 +619,24 @@ private void handleEditorChangeReceived(JSONObject changeMessage) {
         }
 
         /**
+         * Remove a change from the connected UI.
+         */
+        private void handleEditorBindingsCleared(JSONObject clearMessage) {
+            try {
+                final JSONObject payload = clearMessage.getJSONObject("payload");
+                final JSONArray actions = payload.getJSONArray("actions");
+                for (int i = 0; i < actions.length(); i++) {
+                    final String changeId = actions.getString(i);
+                    mEditorChanges.remove(changeId);
```

Comment:
The good news is this isn't an ArrayList, or a list anymore - it's a Map of ids to edits. If the id isn't present, remove returns null and doesn't change the underlying map.